README.rst

```diff
@@ -73,7 +73,7 @@ Project goals
 * Build a Pythonic API for GMT.
 * Interface with the GMT C API directly using ctypes (no system calls).
 * Support for rich display in the Jupyter notebook.
-* Integration with the Scipy stack: numpy.ndarray or pandas.DataFrame for data tables
+* Integration with the SciPy stack: numpy.ndarray or pandas.DataFrame for data tables
```

Comment:
On https://scipy.org/stackspec.html, it says that "The SciPy Stack specification was developed in 2012. As of 2017, the SciPy Stack concept is obsolete given improvements in package management and distribution."

I want to say "PyData stack" instead, but maybe "PyData Ecosystem" is more suitable here? See e.g. https://stackoverflow.com/questions/18168400/the-pydata-ecosystem.

Reply:
"PyData Ecosystem" sounds good.